src/web/command_control/client/components/TaskSettingsPanel.tsx

```diff
@@ -81,6 +83,91 @@ function TaskOptionsPanel(props: Props) {
         props.onChange(newTask)
     }
 
+    /**
+     * Handles the change event for bottom dive parameters.
+     * 
+     * @param {React.ChangeEvent<HTMLInputElement>} evt The event object containing information about the input change.
+     * @returns {void}
+     */
+    function onChangeBottomDiveParameter(evt: React.ChangeEvent<HTMLInputElement>) {
```

Comment:
Is this function needed anymore?

can we just reuse onChangeDriftParameter to update the drift value.